test/sanity/run-test.sh

```diff
@@ -48,4 +48,4 @@ fstype: ext4
 EOF
 
 echo 'Begin to run sanity test for vhd disk feature...'
-"$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --csi.testvolumeparameters="$testvolumeparameters" --ginkgo.skip='should fail when the volume source snapshot is not found|should work|should fail when volume does not exist on the specified path|should fail when the volume does not exist|should fail when the node does not exist|should be idempotent'
+"$CSI_SANITY_BIN" --ginkgo.v --ginkgo.noColor --csi.endpoint="$endpoint" --csi.testvolumeparameters="$testvolumeparameters" --ginkgo.skip='should fail when the volume source snapshot is not found|should work|should fail when volume does not exist on the specified path|should fail when the volume does not exist|should fail when the node does not exist|should be idempotent|should return appropriate capabilities'
```

Comment:
where is the error `should return appropriate capabilities`? may fix it?

Reply:
The error is coming because we are implementing a new capability(Volume_Mount_Group) and this capability isn't recognized by the sanity tests(maybe because it's still an alpha feature). I am not sure if it makes sense to add this new capability to the sanity test considering that it is still an alpha feature. WTYD?

Reply:
only should skip `Node Service NodeGetCapabilities` now in both Line43 & Line51

Reply:
Done